bundles/CoreBundle/src/DependencyInjection/ConfigurationHelper.php

```diff
@@ -47,15 +47,29 @@ public static function addConfigLocationTargetNode(NodeBuilder $node, string $na
             ->addDefaultsIfNotSet()
             ->children()
                 ->arrayNode('write_target')
-                ->addDefaultsIfNotSet()
-                ->children()
-                    ->enumNode('type')
-                        ->values(['symfony-config', 'settings-store', 'disabled'])
-                        ->defaultValue('symfony-config')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->enumNode('type')
+                            ->values(['symfony-config', 'settings-store', 'disabled'])
+                            ->defaultValue('symfony-config')
+                        ->end()
+                        ->arrayNode('options')
+                            ->defaultValue(['directory' => '%kernel.project_dir%' . $folder])
+                            ->variablePrototype()->end()
+                        ->end()
                     ->end()
-                    ->arrayNode('options')
-                        ->defaultValue(['directory' => '%kernel.project_dir%' . $folder])
-                        ->variablePrototype()
+                ->end()
+                ->arrayNode('read_target')
```

Comment:
This is now added to all nodes, but it should be only added to certain nodes, which do not support multiple read targets (such as web to print settings).